biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/TemplateTool.scala

```diff
@@ -17,7 +17,7 @@ package nl.lumc.sasc.biopet.core
 import java.io.{File, PrintWriter}
 
 import nl.lumc.sasc.biopet.utils.summary.Summary
-import nl.lumc.sasc.biopet.utils.{ConfigUtils, Question, ToolCommand}
+import nl.lumc.sasc.biopet.utils.{AbstractOptParser, ConfigUtils, Question, ToolCommand}
```

Comment:
Lots of code duplication in the import statements. Is there an elegant way to make these more nested and hierarchical?
For example: RandomTool imports CommonToolImports. And CommonToolImports imports GeneralImports.